cli/test/encrypt.spec.js

```diff
@@ -22,13 +22,13 @@ let kamusApiScope;
 describe('Encrypt', () => {
   beforeEach(() => {
     sinon.stub(process, 'exit');
-    kamusApiScope = nock(kamusApiUrl)
+    kamusApiScope = nock(kamusUrl)
       .post('/api/v1/encrypt', { data, ['service-account']: serviceAccount, namespace})
       .reply(200, '123ABC');
   });
 
   it('Should return encrypted data', async () => {
-    await encrypt({data, serviceAccount, namespace, kamusApiUrl}, logger);
+    await encrypt(null, {data, serviceAccount, namespace, kamusUrl}, logger);
```

Comment:
why null is added here?

Reply:
`caporal` calls the `action` with 3 parameters: args, options, logger.
I removed the first parameter by mistake because we don't actually use it. But the order matters.